scripts/language_model/word_language_model.py

```diff
@@ -342,8 +357,8 @@ def train():
             L = 0
             with autograd.record():
                 for j, (X, y, h) in enumerate(zip(data_list, target_list, hiddens)):
-                    output, h, encoder_hs, dropped_encoder_hs = forward(X, h)
-                    l = criterion(output, y, encoder_hs, dropped_encoder_hs)
+                    output, h, encoder_hs, dropped_encoder_hs = model.forward(X, h)
```

Comment:
model(X, h)

Reply:
It doesn't work like that, the super object doesn't support forward